tests/load/locustfiles/models.py

```diff
@@ -24,5 +24,15 @@ class RegisterMessage(BaseModel):
 
     messageType: Literal["register"]
     channelID: str
-    status: int
+    status: int = 200
```

Comment:
Instead of an assignment, I think we probably want a literal. Should the register message return without a status an assignment would add a 200, which would give us a false positive.

```suggestion
    status: Literal[200]
```

Reply:
Ah, I didn't mean to include that. Will remove it